test/testvfdswmr.sh.in

```diff
@@ -571,24 +571,21 @@ done
 # read and written by VFD SWMR.
 #
 if [ ${do_zoo:-no} = yes ]; then
-	[ -e ./fifo ] && rm -f ./fifo
-	mkfifo -m 0600 ./fifo
 	rm -f ./shared_tick_num
 	echo launch vfd_swmr_zoo_writer
-	STDIN_PATH="./fifo" catch_out_err_and_rc vfd_swmr_zoo_writer \
-	    ../vfd_swmr_zoo_writer -m 1000 -q &
+	catch_out_err_and_rc vfd_swmr_zoo_writer \
+	    ../vfd_swmr_zoo_writer -q &
 	pid_writer=$!
 
-	STDOUT_PATH="./fifo" catch_out_err_and_rc vfd_swmr_zoo_reader \
-	    ../vfd_swmr_zoo_reader -q -W &
+	catch_out_err_and_rc vfd_swmr_zoo_reader \
+	    ../vfd_swmr_zoo_reader -l 4 -q &
```

Comment:
Better to add a comment "the expected maximal number of ticks for reader to validate"

Reply:
I did it.